cmake/core.cmake

```diff
@@ -23,6 +22,7 @@ target_link_libraries(mbgl-core PRIVATE
 # libraries in our public interface.
 target_link_libraries(mbgl-core PUBLIC
     boost
+    eternal
```

Comment:
Why is this public? It looks like it's only used in the .cpp files.

Reply:
otherwise the `test` target does not compile :/

Reply:
...That sounds odd; it doesn't look like there are any test files referencing eternal or headers that include it?

Reply:
Indeed, it was a leftover in `test/style/source.test.cpp` Thanks for the catch!